parser.y

```diff
@@ -4469,6 +4474,22 @@ ExplainFormatType:
 	{
 		$$ = "json"
 	}
+|	"ROW"
```

Comment:
`ROW` is reserved keywords since 8.0.2, shall we follow it?

https://dev.mysql.com/doc/refman/8.0/en/keywords.html

Reply:
it has been a reserved keyword. i add a test for it.

Reply:
In MySQL8:

```
mysql> create table row (ts varchar(32));
ERROR 1064 (42000): You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'row (ts varchar(32))' at line 1

mysql> create table `row` (ts varchar(32));
Query OK, 0 rows affected (0.02 sec)
```

In 5.7 it is not a reserved keyword, so the behaviour is different.